test/unit/ConfigurationTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit;
+
+use Matthias\SymfonyConfigTest\PhpUnit\ConfigurationTestCaseTrait;
+use OnMoon\OpenApiServerBundle\DependencyInjection\Configuration;
+use PHPUnit\Framework\TestCase;
+
+class ConfigurationTestCase extends TestCase
```

Comment:
There is only one test extending this class, it could use the trait and declare the getConfiguration method itself, and we can get rid of this file.